deployments/capabilities.go

```diff
@@ -19,8 +19,10 @@ import (
 	"path"
 	"strings"
 
+	"fmt"
```

Comment:
General remark here for every one @ystia/yorc-dev: 
It's a best practice to order imports https://github.com/golang/go/wiki/CodeReviewComments#imports

I propose to follow this scheme:

stdlib imports grouped
-a blank line- 
dependencies imports grouped
-a blank line- 
github.com/ystia/yorc/... grouped

This will ease refactoring. For instance migration to go modules required to rename yorc imports and mixed imports was very painful...

Reply:
Maybe we should use goimports instead of/or with gofmt ?
https://godoc.org/golang.org/x/tools/cmd/goimports

Reply:
or use both

Reply:
ok, I add a new issue on this